integration_test/myxql/constraints_test.exs

```diff
@@ -36,20 +114,29 @@ defmodule Ecto.Integration.ConstraintsTest do
   setup_all do
     ExUnit.CaptureLog.capture_log(fn ->
       num = @base_migration + System.unique_integer([:positive])
-      up(PoolRepo, num, ConstraintMigration, log: false)
+      up(PoolRepo, num, ConstraintTableMigration, log: false)
+      up(PoolRepo, num + 1, ProcedureEmulatingConstraintMigration, log: false)
     end)
 
     :ok
   end
 
   @tag :create_constraint
   test "check constraint" do
+    num = @base_migration + System.unique_integer([:positive])
+    ExUnit.CaptureLog.capture_log(fn ->
+      :ok = up(PoolRepo, num, ConstraintMigration, log: false)
+    end)
```

Comment:
other than moving this migration here, the other changes to this test are from `mix format`